### DISCUSSION TITLE
Training with custom arabic data

### DISCUSSION THREAD

**Discussion by alifaqi** [urchade/GLiNER#146] [Q&A] [answered]
Hello, thank you for this great model.
I have a question, suppose I have a custom annotated Arabic dataset, do you think GLINER can be trained with it?
If yes, should i train it from scratch or fine-tune the existing model?

Thanks

**Comment by urchade** [comment]
Sure! Some people have done version for language such as japanese or Korean, on hugging face

**Reply by alifaqi** [reply]
Thank you for your answer. 
Do you think it is better if we train it from scratch or it will be okay to just fine-tune the pre-trained model?

**Reply by urchade** [reply, marked as answer]
It depends on the size of your data, 
If it is > 20k you can train from scratch
